src/content/docs/style-guide/writer-workflow/github-intro.mdx

```diff
@@ -130,9 +130,17 @@ As a **Hero or Sidekick**, make sure you attend to the following throughout your
 * Don't reference internal people by name. If they have a GH account, @mention their GH handle. If they don't, talk instead about teams ("talk to a browser team engineer" or "Support Engineer") rather than people.
 * You can mention the #documentation channel and hero.
 
-## Merge from develop into main work (or, when do we publish?) [#merging-main]
-The Hero currently merges three times a day: At 9 AM, noon, and 3 PM Pacific.
-* Slackbot will remind us about this in #docs_deploys.
+## Merge releases into main work (or, when do we publish?) [#merging-main]
+The Hero currently merges three times a day: At 9 AM (morning), 12 PM (noon), and 3 PM (afternoon) Pacific. We merge release branches into develop to avoid interuptions when someone merges into develop during a release. Read more about this workflow [here](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow).
+
+To start a release:
+
+1. Create a branch based off develop. In Github Desktop, you can do this by clicking **Current branch** in the top header and then selecting develop.
+2. Name the branch following this pattern: `daily-release/mm-dd-yy-time`. Here's an example: `daily-release/10-27-21-morning`. There are branch protections on all branches that follow the `daily-release/*` pattern.
+3. Create a pull request into main from your new daily release branch. You can do this in GitHub Desktop or on Github.com. Pull requests default to merging into develop, so ensure you select main as the `base` branch in the left side of the page.
+4. Wait until all the checks complete, and then merge the pull request.
+
+* Slackbot will remind us about this in #doc_site_deploys.
```

Comment:
Is it worth mentioning this? I either don't need this bullet or I want more info about what Slackbot will remind us about :)